CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 * Fixed an issue where the shields in the instruction are using the style from last navigation session with the `NavigationMapView` injection used in the new session. ([#4197](https://github.com/mapbox/mapbox-navigation-ios/pull/4197))
 * Fixed an issue where the `NavigationMapView.localizeLabels()` method only localized map labels according to the user’s Preferred Language Order setting if the application also had a localization in the preferred language. ([#4205](https://github.com/mapbox/mapbox-navigation-ios/pull/4205))
 * Added `NavigationViewController.annotatesIntersectionsAlongRoute` and `CarPlayNavigationViewController.annotatesIntersectionsAlongRoute` to annotate intersections on the current route step during active navigation. ([#4185](https://github.com/mapbox/mapbox-navigation-ios/pull/4185))
+* Fixed the Route Leg gradient endpoints offset when route lacks traffic congestion data. ([#4217](https://github.com/mapbox/mapbox-navigation-ios/pull/4217))
```

Comment:
Would probably make sense to make this description "less technical". E.g.: `Fixed an issue where the route line with no traffic congestion data and multiple legs wasn't shown correctly`.

Reply:
Thanks for the suggestion, the changeLog is updated.